clr-k8s-examples/setup_system.sh

```diff
@@ -79,6 +80,55 @@ function setup_hosts() {
 	fi
 }
 
+# write increased limits to specified file
+function write_limits_conf() {
+	cat <<EOT | sudo bash -c "cat > $1"
+[Service]
+LimitNOFILE=1048576
+LimitNPROC=1048576
+LimitCORE=1048576
+TimeoutStartSec=0
+MemoryLimit=infinity
+EOT
+}
+
+# update configuration to enable high pod counts
+function config_high_pod_count() {
+	# install bundle dependencies
+	sudo -E swupd bundle-add --quiet jq bc
+
+	# increase max inotify watchers
+	cat <<EOT | sudo bash -c "cat > /etc/sysctl.conf"
+fs.inotify.max_queued_events=1048576
+fs.inotify.max_user_watches=1048576
+fs.inotify.max_user_instances=1048576
+EOT
+	sudo sysctl -p
+
+	# write configuration files
+	sudo mkdir -p /etc/systemd/system/kubelet.service.d
+	write_limits_conf "/etc/systemd/system/kubelet.service.d/limits.conf"
+	if [ "$RUNNER" == "containerd" ]; then
+		sudo mkdir -p /etc/systemd/system/containerd.service.d
+		write_limits_conf "/etc/systemd/system/containerd.service.d/limits.conf"
+	fi
+	if [ "$RUNNER" == "crio" ]; then
+		sudo mkdir -p /etc/systemd/system/crio.service.d
+		write_limits_conf "/etc/systemd/system/crio.service.d/limits.conf"
+	fi
+
+	# increase limits in kubelet
```

Comment:
is /var/lib/kubelet/config.yaml created before cluster init? is it better to propagate via kubeadm.yaml?

Reply:
@dklyle i am already doing monkeying if this kind here. https://github.com/clearlinux/cloud-native-setup/blob/master/clr-k8s-examples/create_stack.sh#L35 😜. I am sure adding `HIGH_POD_COUNT` to that won't hurt us.

Reply:
@krsna1729 I assumed that it was created on install of kubelet. I can verify that, or ...

@ganeshmaharaj my only concern here was having to set the environment variable for two scripts rather than just the one. So I tried to centralize all the work in one script. If we prefer to move this logic to be co-located in create_stack.sh, I can do that.